src/Framework/MSBuildEventSource.cs

```diff
@@ -80,9 +80,9 @@ public void RarComputeClosureStart()
         }
 
         [Event(8, Keywords = Keywords.All)]
```

Comment:
```suggestion
        [Event(8, Keywords = Keywords.All | Keywords.PerformanceLog)]
```

? Or would that not be useful because we'd need Start in there too to get RAR durations?

Reply:
I can put it in for both. I don't know whether we'd need both, but the ones that currently have either with Keywords.PerformanceLog have both annotated that way.

Reply:
By the way, from my perf investigations of yesterday and this morning, it seems like ComputeClosure takes about 3/5 of RAR's overall time.

WriteStateFile took about 5% of its RAROverall time using the BinaryFormatter. It was just over 10% when I did a similar run with Json, which surprised me, since I remembered having verified that System.Text.Json was as fast or faster than BinaryFormatter [here](https://github.com/dotnet/msbuild/pull/5704#discussion_r519069007), but when I reread it, I realized I'd only checked deserialization, and apparently serialization is slower, sadly.